deps.edn

```diff
@@ -401,6 +401,73 @@
 
 ;;; Linters
 
+  ;; clojure -M:kondo --lint src test
+  ;;
+  ;; clojure -M:kondo --version
+  ;;
+  ;; clojure -M:kondo --copy-configs --dependencies --lint "$(clojure -A:dev -Spath)" --skip-lint --parallel
+  ;;
+  ;; Run Kondo from the JVM using the pinned version. Preferable to running the installed command since we can pin the
+  ;; version here which may be different from the version installed on your computer.
+  ;;
+  ;; Use this to only run Kondo against specific files.
+  :kondo
+  {:replace-deps
+   {clj-kondo/clj-kondo {:mvn/version "2024.08.01"}}
+
+   :main-opts
+   ["-m" "clj-kondo.main"]}
+
+  ;; clojure -M:kondo:kondo/all
+  ;;
+  ;; Like the command above but includes arguments so it lints everything.
+  :kondo/all
+  {:main-opts
+   ["-m" "clj-kondo.main"
+    ;; disabled for now since this seems to cause random failures -- see https://github.com/clj-kondo/clj-kondo/issues/2218
+    #_"--parallel"
```

Comment:
Disabled `--parallel` for now since it only saves 5-10 seconds but constantly errors for me -- see https://github.com/clj-kondo/clj-kondo/issues/2218 . This happened before the version bump sometimes but seems to happen even more now. I don't want CI flakes because we were being greedy by trying to save a few seconds.